awx/main/tasks.py

```diff
@@ -1542,9 +1542,11 @@ def run(self, pk, **kwargs):
             status = res.status
             rc = res.rc
 
-            if status == 'timeout':
-                self.instance.job_explanation = "Job terminated due to timeout"
-                status = 'failed'
+            if status in ('timeout', 'error'):
+                self.instance.job_explanation = f"Job terminated due to {status}"
```

Comment:
I just realized that we don't want `job_explanation` to read

> Job terminated due to error

This is particularly bad in the cases where an error occurred and `job_explanation` was set to something in response to that. This code would overwrite that useful message with this vague one.